docs/modules/community/nav.adoc

```diff
@@ -1 +1,6 @@
 * xref:index.adoc[]
+** xref:mailing-lists.adoc[]
+** xref:contributing.adoc[]
+** xref:guidelines.adoc[]
+** xref:download.adoc[]
+** xref:support.adoc[]
```

Comment:
In the nav we should have the following links appearant for ASF compliance:

http://www.apache.org/

http://www.apache.org/foundation/getinvolved.html

http://www.apache.org/foundation/faq.html

http://www.apache.org/licenses/

http://www.apache.org/foundation/sponsorship.html

http://www.apache.org/foundation/thanks.html

http://www.apache.org/security/

Reply:
For the compliance really necessary to put them in nav? Ok now I understand better why it is in the old doc, I found that strange!